### PR TITLE
[IMP] web: SearchPanel in mobile

### PR DESCRIPTION
**Pull request by pparidans**
Enable SearchPanel on mobile.

Task ID: 1917277

### CODE REVIEW

**Comment by pparidans** [comment]
:+1:  @adr-odoo

File: addons/mail/static/src/scss/composer.scss [hunk 1]
```diff
@@ -452,11 +458,15 @@
...
 
         .o_viewer_toolbar {
-            @include o-position-absolute($left: 50%, $bottom: $grid-gutter-width*0.5);
-            transform: translate(-50%, -50%);
+            @include o-position-absolute($bottom: $grid-gutter-width*0.5);
             overflow: hidden;
             border-radius: 4px;
-            background-color: rgba(black, 0.4);
+            justify-content: center;
+            width: 100%;
```
Comment (adr-odoo) [inline addons/mail/static/src/scss/composer.scss]: move on top

**Comment by pparidans** [comment]
:warning: **_to rebase_** :warning:

**Comment by adr-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**